tests/cases/conformance/externalModules/typeOnly/allowsImportingTsExtension.ts

```diff
@@ -0,0 +1,13 @@
+// @allowImportingTsExtensions: false
+// @target: esnext
+// @module: esnext
+
+// @Filename: a.ts
```

Comment:
Is there already an equivalent test case of the 5 assertions below for the case where `a.ts` is `a.d.ts`?

Reply:
I think so, but I'll add one here so it can be compared easily